docs/Index.tsx

```diff
@@ -1,84 +1,80 @@
-import {MultilineInputExamples} from '../src/components/multilineInput/examples/MultilineInputExamples';
-import {NavigationConnectedExamples} from '../src/components/navigation/examples/NavigationConnectedExamples';
-import {NavigationExamples} from '../src/components/navigation/examples/NavigationExamples';
-import {OptionsCycleConnectedExamples} from '../src/components/optionsCycle/examples/OptionsCycleConnectedExamples';
-import {OptionsCycleExamples} from '../src/components/optionsCycle/examples/OptionsCycleExamples';
-import {RadioExamples} from '../src/components/radio/examples/RadioExamples';
-import {MultiSelectExamples} from '../src/components/select/examples/MultiSelectExamples';
-import {SingleSelectExamples} from '../src/components/select/examples/SingleSelectExamples';
-import {SideNavigationLoadingExample} from '../src/components/sideNavigation/examples/SideNavigationLoadingExample';
-import {SliderExamples} from '../src/components/slider/examples/SliderExamples';
-import {SplitLayoutExamples} from '../src/components/splitlayout/examples/SplitLayoutExamples';
-import {StepProgressBarExamples} from '../src/components/stepProgressBar/examples/StepProgressBarExamples';
-import {SubNavigationConnectedExamples} from '../src/components/subNavigation/examples/SubNavigationConnectedExamples';
-import {SubNavigationExamples} from '../src/components/subNavigation/examples/SubNavigationExamples';
-import {LinkSvgExamples} from '../src/components/svg/examples/LinkSvgExamples';
-import {SvgExamples} from '../src/components/svg/examples/SvgExamples';
-import {SyncFeedbackExample} from '../src/components/syncFeedback/examples/SyncFeedbackExample';
-import {TabsExamples} from '../src/components/tab/examples/TabConnectedExample';
-import {TableWithDisabledRowsExamples} from '../src/components/tables/examples/TableDisabledRowsExamples';
-import {TableEmptyRowExamples} from '../src/components/tables/examples/TableEmptyRowExamples';
-import {TableExamples} from '../src/components/tables/examples/TableExamples';
-import {TableHeaderExamples} from '../src/components/tables/examples/TableHeaderExamples';
-import {TableRowConnectedExamples} from '../src/components/tables/examples/TableRowConnectedExamples';
-import {TableRowExamples} from '../src/components/tables/examples/TableRowExamples';
-import {TitleExamples} from '../src/components/title/examples/TitleExamples';
-import {ToastConnectedExamples} from '../src/components/toast/examples/ToastConnectedExamples';
-import {ToastExamples} from '../src/components/toast/examples/ToastExamples';
-import {TooltipExamples} from '../src/components/tooltip/examples/TooltipExamples';
-import {UserFeedbackExample} from '../src/components/userFeedback/examples/UserFeedbackExample';
-import {MembersExample} from './members-example/MembersExample';
-import {ReactVaporStore} from './ReactVaporStore';
+import { render as ReactDOMRender } from 'react-dom';
```

Comment:
I think you should revert this change. We're "fighting" for the spaces in every PR which is annoying

Reply:
After merging with master and rebuilding the project, if you run `npm run lintfix` it should fix all this.